packages/grafana-schema/src/common/common.gen.ts

```diff
@@ -642,6 +645,14 @@ export interface TableBarGaugeCellOptions {
   type: TableCellDisplayMode.Gauge;
 }
 
+/**
+ * Area chart cell options
+ */
+export interface TableAreaChartCellOptions extends GraphFieldConfig {
+  color?: string;
```

Comment:
why this option?? we already have a standard field option for color

Reply:
2 reasons:

Intention here is to just allow setting single fixed color. `FieldColor` type & config have color scheme, where fixed color is just one of the options. Seems overkill as currently sparkline only supports a single series. It also allows selecting thresholds based scheme, but haven't added thresholds support to sparkline yet, and unsure if it should be added.

`GraphFieldConfig` does not include `color` property, currently it's a property of `FieldConfig`. Both `FieldConfig` and `FieldColor` are currently marked private in cue, can't use them for `TableAreaChartCellOptions` without bigger changes to cue definitions.

So I thought I could get away with making this simpler :)

Reply:
:man_facepalming:  I see what you mean now, standard options -> color scheme. Nevermind, will remove this

Reply:
done